src/modules/person/domain/person.ts

```diff
@@ -79,6 +80,7 @@ export class Person<WasPersisted extends boolean> {
         public isLocked: boolean | undefined,
         public email: string | undefined,
         public oxUserId: string | undefined,
+        public istTechnisch: boolean,
     ) {
         this.mandant = Person.CREATE_PERSON_DTO_MANDANT_UUID;
     }
```

Comment:
nicht nötig. Der Sichtbarkeitsmodifikator `public` in Zeile 83. Sorgt schon dafür.